dev-guide/cluster-fleet-evaluation.md

```diff
@@ -0,0 +1,160 @@
+---
+title: cluster-fleet-evaluation
+authors:
+  - "@aravindhp"
+reviewers:
+  - "@openshift/openshift-patch-managers"
+approvers:
+  - "@openshift/openshift-patch-managers"
+creation-date: 2022-08-16
+last-updated: 2022-08-16
+---
+
+# Cluster Fleet Evaluation
+
+## Problem Statement
+
+OpenShift components sometimes need to introduce a change that could result in
+highly visible alerts, breakages or upgrade failures on user clusters. In
+scenarios like this it will be useful for the component owner to assess how many
+customers would be affected before rolling out the change and based on that
+feedback decide on the level of investment required to properly inform the
+customer base, or change implementation details.
+
+## Solution
+
+Rather than introducing the change, add just the detection logic to the operator
+for the change and associate it with a
+[ClusterStatusConditionTypes](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L145)
+called _ClusterFleetEvaluation_. If the
+[ConditionStatus](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L124)/
+[Reason](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L133)
+associated with this type is
+[ConditionTrue](https://github.com/openshift/api/blob/cc0db1116639638254e87a564902833f1ee006d5/config/v1/types_cluster_operator.go#L107)
+or matches a new _ConditionStatus/Reason_ you are expecting, it will indicate
+that the change would have affected that cluster. You can join multiple
+_ConditionStatus/Reason_ to test multiple scenarios. Now release the detection
+logic to master and
+[backport](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit)
+it to a N-1 Z stream release where N is the current release under development.
+The Z stream backport is useful as it allows the team to gather the data without
+going through a full release cycle. The backport should follow the
+[OpenShift backport process](https://docs.google.com/document/d/1PC87sSFa_zGCk95kXDW-wrVxnlgBmkHqpOgQnd4bbUw/edit).
```

Comment:
nit: probably no need to link the backport process doc twice.  If we keep this stand-alone sentence here, dropping the link from `Now release the detection logic to master and backport it...` above would avoid readers possibly clicking through twice and then thinking "oh, wait, I just opened a tab for this.  No need for a second tab for the same content...".